README.md

```diff
@@ -66,9 +66,11 @@ yarn proofs:update
 
 ## 📚 Documentation
 
-ℹ️ Check the [official Cartesi Rollups documentation website](https://docs.cartesi.io/cartesi-rollups/overview/).
+🚀 Smart contract documentation is kept up-to-date and deployed [**here**](https://cartesi.github.io/rollups-contracts).
```

Comment:
is this the correct url?

Reply:
Yes, it will be live as soon as this gets merged and runs for the first time.